intermediate/challenge-1/.env

```diff
@@ -0,0 +1,14 @@
+PROJECT_NAME=Audela_Challenge
+API_HOST=0.0.0.0
+API_PORT=8080
+POSTGRES_SERVER=postgresDB
+POSTGRES_USER=postgres
+POSTGRES_DB=Audela
+POSTGRES_PASSWORD=123456789
+FIRST_MANAGER=firsr_manager@audela.com

```

Comment:
It seems more appropriate to avoid using the environment variables for storing the test-data for 'FIRST_MANAGER', instead these values can belong in Database Setup and/or test logic.

Reply:
To keep sensitive data safe, I think to create the first user with a password (FIRST_MANAGER and FIRST_MANAGER_PASSWORD), it's better to use environment variable references instead of setting passwords directly in the configuration file or the database configuration. These are set by who’s responsible for the deployment environment.

for the EMAIL_TEST_USER variable which was needed for testing, yes it's more appropriate to separate it from the config.py file and provide it via a test config file with a database configuration in a controlled testing environment.